selene_sdk/utils/performance_metrics.py

```diff
@@ -420,8 +435,13 @@ def update(self, prediction, target, target_mask=None):
         """
         metric_scores = {}
         for name, metric in self.metrics.items():
+            if metric.transform is not None:
+                tr_prediction, tr_target, tr_target_mask = metric.transform((prediction, target, target_mask))
+            else:
+                tr_prediction, tr_target, tr_target_mask = prediction, target, target_mask
+            assert tr_prediction.shape == tr_target.shape == tr_target_mask.shape
```

Comment:
```suggestion

```
While this should hold for the cases we've had so far, we should either remove it here or I'll have to delete this line in further commits, because this assertion would not be expected to pass in the case of sequential targets, for which these shapes will be different. We'll have prediction and target shapes `(batch_size, target_length, n_features)`, but mask shape `(batch_size, n_features)`. Or in the case of `multi_ct` model predictions of shape `(batch_size, n_cell_types, target_length, n_features)` and masks of shape `(batch_size, n_cell_types, n_features)`.
I could add an extra dimension of `target_length` to masks for the sake of consistency, but that would be an unnecessary repetition taking up RAM